staker-rewards/lib/requirements.js

```diff
@@ -179,16 +229,58 @@ export default class Requirements {
   async wasSortitionPoolDeauthorized(operator) {
     return this.operatorsDeauthorizedInInterval.has(operator.toLowerCase())
   }
+
+  async checkMinimumStakeAtIntervalStart(operator) {
+    return await callWithRetry(
+      this.bondedECDSAKeepFactory.methods.hasMinimumStake(operator),
+      this.interval.startBlock
+    )
+  }
+
+  // If the operator has at least minimum unbonded value available they have
+  // to be registered in the sortition pool. Operators who are not in the sortition
+  // pool because all of their ether is locked are still getting rewards because
```

Comment:
Let's change "all of their ether is locked" to "all of their ether is bonded" because we define ETH_locked as ETH_bonded + ETH_unbonded in other places in staker-rewards scripts.